tests/integration_tests/reports/api_tests.py

```diff
@@ -52,17 +53,67 @@
 
 class TestReportSchedulesApi(SupersetTestCase):
     @pytest.fixture()
-    def create_working_report_schedule(self):
+    def create_working_admin_report_schedule(self):
         with self.create_app().app_context():
 
             admin_user = self.get_user("admin")
```

Comment:
one nit, in order to avoid extra indeed and context grammar, we should use `app_context` and `login_admin` fixtures

```python
def create_working_report_schedule(app_context, login_admin):
    ....
    ....

```

Reply:
For some reason I'm having trouble using these fixtures with the class-based tests. Let's leave this for a follow-up refactor PR when these tests are refactored into functional tests.